### DISCUSSION TITLE
Update from version 4.0.28 to 4.0.84 doesn't work

### DISCUSSION THREAD

**Discussion by marionet** [Dan-in-CA/SIP#255] [Q&A]
![image](https://user-images.githubusercontent.com/4542954/115118065-d0689d00-9fa1-11eb-9dd1-2feddb3d284d.png)

Hello Dan.

When clicking on UPDATE SIP. I get the notice: SIP is restarting. This may take several seconds....
But afterwards everything is still the same. Also restart does not make a difference.
What could be the problem?

Best regards,
Mario.

**Comment by Dan-in-CA** [comment]
If the system update plugin does not work you can update from the command line.
Log into the SIP directory and use the command:
`git pull`

That should update sip to the latest version.
Dan

**Comment by marionet** [comment]
Thank you Dan.

If the system update plugin does not work you can update from the command line.
Log into the SIP directory and use the command:
git pull

That should update sip to the latest version.
Dan
